src/nurse/management/commands/_notifications.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from dotenv import load_dotenv
+from onesignal_sdk.client import Client
+
+load_dotenv()
+
+app_id = os.environ.get("ONESIGNAL_APP_ID")
+api_key = os.environ.get("API_KEY")
```

Comment:
Actually let's put that to `src/main/settings.py`, make a section just below `django-templated-mail` config.
Then you can import and use the settings with:
```python
from django.conf import settings
app_id = settings.ONESIGNAL_APP_ID
api_key = settings.ONESIGNAL_API_KEY
```

Reply:
ok